airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py

```diff
@@ -126,6 +126,17 @@ def path(
     ) -> str:
         return "ledger/account"
 
+class BalanceSheet(TripletexApiStream):
+    primary_key = "id"
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
+
+    def path(
+            self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return "balanceSheet"
+
 
 class SourceTripletexApi(AbstractSource):
```

Comment:
You need to register a new stream by adding a corresponding class to the return statement. 
Example: `[Postings(config=config), Departments(config=config), Accounts(config=config), BalanceSheet(config=config)]`